src/controller/LottoControllerWeb.js

```diff
@@ -0,0 +1,86 @@
+import LottoMachine from '../domain/LottoMachine';
+import Money from '../domain/Money';
+import calculateROI from '../util/calculateROI';
+import statisticsModal from '../../static/html/statisticsModal';
+import purchaseAfter from '../../static/html/purchaseAfter';
+import purchaseBefore from '../../static/html/purchaseBefore';
+import onError from '../util/onError';
+import renderUI from '../util/renderUI';
+import findErrorAndRemove from '../util/findErrorAndRemove';
+import closeModal from '../util/closeModal';
+
+renderUI('purchase-before', 'afterbegin', purchaseBefore());
+
+const purchaseAmount = document.querySelector('.lp-pa-input-group');
+const purchaseAmountButton = document.querySelector('.lp-pa-input-btn');
+const findPurchaseAfter = document.getElementById('purchase-after');
+
+purchaseAmountButton.addEventListener('click', (event) => {
+  event.preventDefault();
+  const inputMoney = document.querySelector('.lp-pa-input-amount').value;
+  try {
+    const money = new Money(Number(inputMoney));
+    const lottoMachine = new LottoMachine(money.count);
+    findErrorAndRemove();
+    printPurchaseAfter(findPurchaseAfter, money.count, lottoMachine.lottos);
+    handleClickLottoResult(lottoMachine, money);
+  } catch (error) {
+    onError(purchaseAmount, error.message);
+  }
+});
+
+function handleClickLottoResult(lottoMachine, money) {
+  const printLottoResultButton = document.querySelector('.lp-winning-floating-btn');
+  printLottoResultButton.addEventListener('click', (event) => {
+    event.preventDefault();
+    executionWinningAndBonusNumbers(lottoMachine, money);
+  });
+}
+
+// TODO: 에러 나는 부분 빨간 박스 처리해주기
+// TODO: 로또 출력 많을 때, 더보기 버튼으로 접었다 폈다 하기
+// TODO: 당첨 번호 입력할 때, autoFocusing 및 하나 입력하면 다음 input으로 포커싱해주기
+function executionWinningAndBonusNumbers(lottoMachine, money) {
+  const winningBonusNumbersGroup = document.querySelector('.lp-nig-number-input-group');
+  try {
+    inputWinningAndBounusNumbers(lottoMachine);
+    findErrorAndRemove();
+    printLottoResultsAndProfitRate(lottoMachine, money);
+    closeModal('lm-close-btn');
+    closeModal('modal-body');
+    retryLotto();
+  } catch (error) {
+    onError(winningBonusNumbersGroup, error.message);
+  }
+}
+
+function printLottoResultsAndProfitRate(lottoMachine, money) {
+  const lottoRanks = lottoMachine.countLottoRanks();
+  const totalProfitRate = calculateROI(money, lottoRanks);
+  renderUI('statistics-modal', 'beforebegin', statisticsModal(lottoRanks, totalProfitRate));
+}
+
+function inputWinningAndBounusNumbers(lottoMachine) {
```

Comment:
오타 발견!
```suggestion
function inputWinningAndBonusNumbers(lottoMachine) {
```
오타를 체크해주는 vscode 익스텐션을 추천드립니다ㅎㅎ 
- https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker